src/preamble.js

```diff
@@ -595,9 +595,11 @@ var wasmBinaryFile;
 if (Module['locateFile']) {
 #endif
   wasmBinaryFile = '{{{ WASM_BINARY_FILE }}}';
+#if !SINGLE_FILE
```

Comment:
I suppose this is because we know a priori that `wasmBinaryFile` is a data URI in this mode?

Reply:
Does this not change any of our code size expectations?

Reply:
> I suppose this is because we know a priori that `wasmBinaryFile` is a data URI in this mode?

Yeah, that's right. In `SINGLE_FILE` builds the Wasm content is embedded, so `locateFile` is not needed.

> Does this not change any of our code size expectations?

Looks like not, CI shows green.